olive/passes/pytorch/lora.py

```diff
@@ -588,6 +588,7 @@ def train_and_save_new_model(
         :param output_model_path: The path to save the output model to.
         :return: The output model handler.
         """
+        allow_tf32 = None
```

Comment:
we can ignore the warning here since allow_tf32 is only defined for cuda.

Reply:
This kind of isssues are shown as `errors`.

Reply:
I mean we might need find some ways to ignore/disable this errors since we might not be able fix all errors.